recipes/unique-uncertainty/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "unique-uncertainty" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/unique_uncertainty-{{ version }}.tar.gz
+  sha256: b7d22a091e09081e9af6e6e7e55d301eacbd258cf8fdd9109845f33bb55b7792
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<=3.12.1
+    - flit-core >=3.2,<4
+    - pip
+  run:
+    - python >=3.8,<=3.12.1
+    - colorcet >=3.0,<=3.0.1
+    - ipykernel >=6.29,<=6.29.2
+    - jinja2 >=3.1,<=3.1.3
+    - matplotlib-base >=3.6,<=3.8.2
+    - numpy >=1.19,<=1.26.4
+    - pandas >=2.0,<=2.2.0
+    - pyarrow <=16.1.0
+    - pyyaml >=6.0,<=6.0.1
+    - rdkit ==2023.09.1
+    - ruamel.yaml >=0.17
+    - scikit-learn >=1.3,<=1.4.0
+    - scipy >=1.10,<=1.12.0
+    - seaborn >=0.13,<=0.13.2
+    - shapely >=2.0,<=2.0.2
+    - statsmodels >=0.13.3,<=0.14.2
+    - tqdm >=4.65,<=4.66.2
+
+test:
+  imports:
+    - unique
+    - unique.pipeline
+  commands:
+    - pip install rdkit==2023.09.1
```

Comment:
This is not recommended! If you are doing this just to pass pip-check, please note the issue with the check and comment it out.

Reply:
Hi @ocefpaf,

Thank you for the help! 

I tried this hack-y solution because the `pip check` command was raising the following error (e.g., see [L1104](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1037984&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1104) of the `linux_64` job - though the same error happens in `osx osx_64` and `win_64 win` jobs):

```bash
unique-uncertainty 0.2.0 requires rdkit, which is not installed.
```

However, you can see in the [previous lines that actually the `rdkit` package is correctly installed](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1037984&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1043) in the environment.

Furthermore, the `rdkit` dependency is [explicitly stated in the recipe](https://github.com/mtdhuynh/staged-recipes/blob/711b4d03f6f5339641f85bb468ec111ac0b4590b/recipes/unique-uncertainty/meta.yaml#L32C12-L32C13).

I saw that this issues might be due to conflicting/different names between `pypi` and `conda` distributions (see this [SO answer](https://stackoverflow.com/a/71787513)), but the `rdkit` package has the same naming in both [`pypi`](https://pypi.org/project/rdkit/) and [`conda`](https://anaconda.org/conda-forge/rdkit).

Could you provide further delucidation on this matter? Thanks!

Reply:
This is a corner case, not uncommon, where it is fine to comment the pip-check out. Just add some text above it explaining why. But it is better to not perform pip-check, than risking a faulty test against a PyPI dependency.

Reply:
Thank you @ocefpaf for the prompt help and advice! I have commented it out and now all checks pass.